src/Type.zig

```diff
@@ -97,7 +99,7 @@ pub const Func = struct {
     params: []Param,
 
     pub const Param = struct {
-        name: []const u8,
+        name: StringId,
```

Comment:
You might want to try to reorganize these fields for better memory usage now that the name fields take up less space.